src/app/page/categoryInfo/course/course.component.html

```diff
@@ -9,10 +9,24 @@
         <star-rating-comp [starType]="'svg'" [showHalfStars]="true"
                           [rating]="course.rating" (onClick)="onRatingClick(course, $event)"></star-rating-comp>
       </td>
-      <ng-template #unAuthorized>
+            <ng-template #unAuthorized>
         <star-rating-comp [starType]="'svg'" [showHalfStars]="true"
                           [rating]="course.rating" [readOnly]="true"></star-rating-comp>
       </ng-template>
+      <td class="alignCenter">
+        <div *ngIf="!course.status">
+          <div (click)="addCourseToUser(course.courseId)" *ngIf="isAuthorized()"
+               class="button addDeckToUserButtonActive"
```

Comment:
please rename class addDeckToUserButtonActive, because it is used for decks and for courses.

Reply:
Done. :)